pkg/virtual/workspaces/registry/rest.go

```diff
@@ -678,28 +682,14 @@ func (s *REST) Delete(ctx context.Context, name string, deleteValidation rest.Va
 		Name:            internalName,
 		ResourceRequest: true,
 	}
-	if decision, _, err := authz.Authorize(ctx, deleteWorkspaceAttr); err != nil {
+	decision, reason, err := authz.Authorize(ctx, deleteWorkspaceAttr)
+	if err != nil {
 		klog.Errorf("failed to authorize user %q to %q clusterworkspaces/workspace name %q in %s", userInfo.GetName(), "delete", internalName, orgClusterName)
 		return nil, false, kerrors.NewForbidden(tenancyv1beta1.Resource("workspaces"), "", fmt.Errorf("deletion in workspace %q is not allowed", orgClusterName))
 	} else if decision != authorizer.DecisionAllow {
-		// check for admin verb on the content
```

Comment:
Do you want to comment this out instead of deleting it? I imagine we'll want to reenable it once we have proper delete?

Reply:
Yeah I can just comment out.